### DISCUSSION TITLE
MOOSE Workshop June 6-8

### DISCUSSION THREAD

**Discussion by GiudGiud** [idaholab/moose#23991] [News]
Hi everyone,

We will be holding a MOOSE training/workshop June 6-8 in Idaho Falls.
INL has negotiated discounted hotel rates. There will also be a virtual option to attend.

Please register here:
https://inlhrfedramp.gov1.qualtrics.com/jfe/form/SV_1RDZf244xsF1NK6

Guillaume for the MOOSE team

**Comment by stu314159** [comment]
Dear Moose Team,

Are Workshop slots still available?  I attempted to follow the link and was presented this message:

![image](https://user-images.githubusercontent.com/2084562/235457200-85d028e5-e8f3-431e-b451-80f706a7fa3e.png)

**Reply by GiudGiud** [reply]
I think this is not intentional
we ll fix this and get back to you

**Reply by permcody** [reply]
The link is working again. Please register!

**Reply by stu314159** [reply]
Thank you!

**Reply by GiudGiud** [reply]
Link is broken again, will keep you posted!

**Reply by salaudeen-ya** [reply]
I just register now. I think it's fixed.

**Reply by GiudGiud** [reply]
nice

**Comment by ecmLab** [comment]
I can only attend virtually; is there any agenda available already? Thanks!

**Reply by GiudGiud** [reply]
Agenda is not available yet. Please register for virtual attendance and it will be sent as soon as we have it

**Comment by alexlanl** [comment]
Hello!
Sorry, could you please update the registration link again? It doesn’t work. 
Thank you!

**Comment by permcody** [comment]
I’ll see if we can get any hotel deals set up next week! Last year we
weren’t able to do much because we didn’t have a large enough block. As
Guillaume said, it’s probably best to book now. If you’ve registered, you
are in. We will cut off registration completely when we are fully booked.

Cody
